neqo-http3/src/control_stream_local.rs

```diff
@@ -0,0 +1,57 @@
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+use crate::hframe::HFrame;
+use crate::Res;
+use neqo_common::{qdebug, Encoder};
+use neqo_transport::{Connection, StreamType};
+
+pub const HTTP3_UNI_STREAM_TYPE_CONTROL: u64 = 0x0;
+
+// The local control stream, responsible for encoding frames and sending them
+#[derive(Default, Debug)]
+pub struct ControlStreamLocal {
+    stream_id: Option<u64>,
+    buf: Vec<u8>,
+}
+
+impl ::std::fmt::Display for ControlStreamLocal {
+    fn fmt(&self, f: &mut ::std::fmt::Formatter) -> ::std::fmt::Result {
+        write!(f, "Local control stream {:?}", self.stream_id)
+    }
+}
+
+impl ControlStreamLocal {
+    pub fn send_frame(&mut self, f: HFrame) {
```

Comment:
doesn't actually send, it enqueues.

Reply:
Would this be more correct as `queue_frame`? Not that this deserves a lot of back-and-forth, but let's just spend a minute to see if a better name is out there.

Reply:
I called it enqueue_frame, but queue_frame is fine too.